int32 class_id
string class_name
float64 score

int32[] bbox

# location
Point position

# dimension
Vector3 dimensions

float64 alpha

float64 theta_ray

float64[] bin_anchor

float64[] bin_confidence